docs/contributing/first-time.md

```diff
@@ -168,15 +197,26 @@ Once you have completed, tested, and linted your code, and created a {ref}`contr
 
 1.  Visit your fork of the Plone repository on GitHub, and [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) against the development branch.
     -   Make your title and description descriptive.
-        Don't be lazy with "Fixes bug" or other useless drivel.
-    -   To automatically close the original issue when your pull request is merged, include "Closes #issue_number" in your description.
+    -   Include "Fixes #" and the related issue number.
+        This enables automatic closing of the related issue when the pull request is merged.
         This also creates a hyperlink to the original issue for easy reference.
+    -   Don't be lazy with "Fixes bug" or other useless drivel.
 1.  [Request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from other team members.
 1.  Members who subscribe to the repository will receive a notification and review your request.
 
-```{todo}
-Provide instructions when the contributor needs to update their pull request with changes from the default branch.
-Members of Contributors do not have the button "Update branch" to easily do this, and must use git foo to manage the situation.
+
+### Update your pull request from your fork
```

Comment:
I don't think these instructions will do the right thing when working from a fork. It will pull the main branch from the fork, but not update it from the upstream repository.

Reply:
Oh, that's right, unless we track the upstream, which we should do, too. Pull from upstream, push to origin. I forgot that, being a privileged member.

Reply:
@davisagli I made a couple of revisions. Would you please take another look to verify my sad gitfu?

Reply:
@stevepiercy That's better. I'm making a couple more suggestions.